operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/config/ConfigurationService.java

```diff
@@ -118,4 +118,8 @@ default Metrics getMetrics() {
   default ExecutorService getExecutorService() {
     return Executors.newFixedThreadPool(concurrentReconciliationThreads());
   }
+
+  default boolean getCloseClientOnStop() {
```

Comment:
How about just `closeClientOnStop`? I always find it weird to see `getDoSomething` methods :)

Reply:
agree, changed